src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -174,6 +179,90 @@ private BoundAttribute BindAttributeCore(AttributeSyntax node, NamedTypeSymbol a
                 boundNamedArguments, resultKind, attributeType, hasErrors: resultKind != LookupResultKind.Viable);
         }
 
+        private void CheckArgumentNullability(MethodSymbol attributeConstructor, AnalyzedAttributeArguments arguments, DiagnosticBag diagnostics)
+        {
+            // Check constructor arguments
+            for (int i = 0; i < arguments.ConstructorArguments.Arguments.Count; i++)
+            {
+                var argument = arguments.ConstructorArguments.Argument(i);
+                var argName = arguments.ConstructorArguments.Name(i);
+
+                // map the argument back to the correct parameter
+                var parameter = argName == null
+                                ? attributeConstructor.Parameters[i]
```

Comment:
>i [](start = 66, length = 1)

Are we handling the `params` array case where there are more arguments than parameters?